infrastructure/channels/serialize_child.html

```diff
@@ -0,0 +1,167 @@
+<!doctype html>
+<script src="/resources/channel.sub.js"></script>
+<script src="serialize-data.js"></script>
+<script>
+
+let lastData;
+
+// Hack: these will be recreated as assert failures in the caller
+function AssertionError(message) {
+    this.message = message;
+}
+AssertionError.prototype = Object.create(Error.prototype);
+
+function compareResult(name, actual) {
+    let obj = objects[name]
+    let expected = obj.hasOwnProperty("output") ? obj.output : obj.input;
```

Comment:
Why do we need to ensure that `output` is an own property, but don't care for `input`?